charts/guac/README.md

```diff
@@ -68,18 +69,18 @@ See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_h
 
 
 ### Accessing GUAC 
-GUAC exposes a few interfaces for user interaction - including the GraphQL Server/Playground, and the Visualizer. You can access them via ```kubectl port-forward```
+GUAC exposes a few interfaces for user interaction - including the GraphQL Server/Playground, NATS, and CollectSub service. You can access them via ```kubectl port-forward```
 ```
 kubectl port-forward svc/graphql-server 8080:8080
-kubectl port-forward svc/visualizer 3000:3000 
-```
-
-If there is a need to access the NATS and CollectSub service, you can run
-```
 kubectl port-forward svc/guac-nats 4222:4222
 kubectl port-forward svc/collectsub 2782:2782
 ```
 
+### GUAC Visualizer
+The GUAC visualizer can be set up following the documentation at https://docs.guac.sh/guac-visualizer/. 
```

Comment:
same here for `https://docs.guac.sh/guac-visualizer/`

Reply:
will incorporate the change in next PR when I add a test workflow